### ISSUE TITLE
Create "Lambda Cartel" marketing ad.

### ISSUE THREAD

**Issue by bizmonger** [Lambda-Cartel/everything#16]
As a marketer, I want to send Lambda Cartel marketing ads via periodic tweets and LinkedIn posts.

**Comment by yelluw** [comment]
Yes, there is a need for a media strategy. Lets define goals first and then define milestones.

Goals are what we aim to get out of the effort.

Milestones are the steps we need to take to reach the goals.

Things might not be very clear right now because we are still figuring out other things. Lets use this issue to air out all thoughts about this.
